group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorServiceTest.java

```diff
@@ -4409,89 +4335,6 @@ public void testDescribeShareGroupAllOffsetsReadSummaryPartitionError() throws I
         assertEquals(responseData, future.get());
     }
 
-    @Test
-    public void testDescribeShareGroupAllOffsetsLatestOffsetError() throws InterruptedException, ExecutionException {
```

Comment:
You should have either of tests to make sure the handling in GroupCoordinatorService is done right if there happens to be a future compted exceptionally.